src/components/Request/FormFields/Supervisory.tsx

```diff
@@ -0,0 +1,24 @@
+import { Radio } from "@fluentui/react-components";
+import "components/Request/Request.css";
+import { RHFRequest } from "components/Request/NewRequestForm";
+import BACRadioGroup from "components/BaseFormFields/BACRadioGroup";
+
+const Supervisory = () => {
+  return (
+    <div className="requestFieldContainer">
+      <BACRadioGroup<RHFRequest>
+        name="supervisory"
+        labelText="Supervisory Position"
```

Comment:
Should all places this appears be called Supervisory/Lead Position, according to AC1?